api/config/swagger.py

```diff
@@ -415,6 +444,37 @@ def schema_view_swagger(request):
                     ],
                     description='Fetching questions on a specific meetup'
                 ),
+                'update existing answer': coreapi.Link(
+                    url='/api/meetups/{meetupId}/questions/{questionId}/answers/{answerId}',
+                    action='PUT',
+                    fields=[
+                        coreapi.Field(
+                            name='questionId',
```

Comment:
can you change the name form questionid to meetupid? since you have to two fields with the same name

Reply:
Made the requested changes